framework/src/org/checkerframework/framework/type/treeannotator/ImplicitsTreeAnnotator.java

```diff
@@ -104,6 +118,20 @@ public void addTreeClass(Class<? extends Tree> treeClass, AnnotationMirror theQu
                     treeClasses + " at " + treeClass + " with " +theQual);
         }
     }
+    public void addLiteralKind(LiteralKind literalKind, AnnotationMirror theQual) {
```

Comment:
A brief Javadoc comment, for this method and the one above it, would be helpful.